coord/src/rmr/redise_parser/grammar.c

```diff
@@ -50,7 +50,7 @@ void yyerror(char *s);
 
 static void parseCtx_Free(parseCtx *ctx) {
     if (ctx->my_id) {
-        free(ctx->my_id);
+        rm_free(ctx->my_id);
```

Comment:
This is a temp allocation during parsing and it is generated from the grammar - no need to change

Reply:
@oshadmi  Why we don't want to count temporary allocation?
You said we want redis to cap even temporary (read) allocations.

Reply:
@OfirMos If the lifetime of the allocations within the parser is only during parsing the query expression, and once the corresponding QueryNode structs are created (using rm_ allocation), then the parser memory is deallocated, we can skip tracking it since it is just a temp step on the main thread, which cannot be reflected to the user.

@MeirShpilraien WDYT?